src/main/java/edu/ksu/canvas/errors/ErrorHandler.java

```diff
@@ -1,14 +1,14 @@
 package edu.ksu.canvas.errors;
 
-import org.apache.http.HttpRequest;
-import org.apache.http.HttpResponse;
+import org.apache.hc.core5.http.ClassicHttpRequest;
+import org.apache.hc.core5.http.ClassicHttpResponse;
 
 /**
  * This allows additional specific behaviour for handling errors.
  */
 public interface ErrorHandler {
 
-    boolean shouldHandle(HttpRequest httpRequest, HttpResponse httpResponse);
+    boolean shouldHandle(ClassicHttpRequest httpRequest, ClassicHttpResponse httpResponse);
```

Comment:
So this is the change you mentioned.  Since this handled entities, this would be needed.

Reply:
if you check the previous commits you'll see what I mean